pkg/daemon/ceph/client/filesystem.go

```diff
@@ -169,17 +170,9 @@ func IsMultiFSEnabled() bool {
 }
 
 // SetNumMDSRanks sets the number of mds ranks (max_mds) for a Ceph filesystem.
-func SetNumMDSRanks(context *clusterd.Context, clusterName, fsName string, activeMDSCount int32) error {
-	// Noted sections 1 and 2 are necessary for reducing max_mds in Luminous.
```

Comment:
Please leave these notes minus the TODO item. These comments are helpful for understanding why this code exists and inform when it may be removed later.